### DISCUSSION TITLE
[@types/video.js] Missing function signature of `extend` for video.js

### DISCUSSION THREAD

**Discussion by tomasfabian** [DefinitelyTyped/DefinitelyTyped#57267] [Issues with a @types package]
File [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/video.js/index.d.ts) should include function signature of [extend](https://docs.videojs.com/module-extend.html):

```JavaScript
var MyComponent = videojs.extend(videojs.getComponent('Component'), {
  myCustomMethod: function() {
  }
});
```

Thank you

[a comment by an automated account is not shown]

**Comment by joeflateau** [comment]
@tomasfabian will you send a PR for this?

**Comment by tomasfabian** [comment]
Hi @joeflateau,
I've just jumped into TypeScript and I was not able to figure out the correct TS parameters and I think that it could be done better, so I'm rather won't send a PR (I apologize):

```TypeScript
function extend(superClass: any, subClassMethods?: any): Component;
``` 

Thank you.

**Comment by joeflateau** [comment]
@tomasfabian no worries, i'll take a look
